src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
+  Triangle = 'triangle',
+  Cicle = 'circle',
+  Rectangle = 'rectangle',
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  public shape;
```

Comment:
Why do you need this and 32 line?

You could simply write here 👇 
```suggestion
  shape = Shape.Triangle;
```